lib/ViewModels/GazetteerSearchProviderViewModel.js

```diff
@@ -113,6 +114,24 @@ GazetteerSearchProviderViewModel.prototype.search = function(searchText) {
     });
 };
 
+// Given a list of results sorted in decreasing importance, strip results that are close to another result with the same name
+function stripDuplicates(results) {
+    var i;
+    var placeshash={};
+    var stripped = [];
+    for (i=0; i < results.length; i++) {
+        var lat = Number(results[i].location.split(',')[0]).toFixed(1);
```

Comment:
Is the result much worse if we use, say, 0.01 degrees instead of 0.1?  20km seems far enough apart that it could have meaningfully distinct results with similar names within that distance.